actors/power/src/types.rs

```diff
@@ -55,6 +57,12 @@ pub struct EnrollCronEventParams {
     pub payload: RawBytes,
 }
 
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone, Default)]
+#[serde(transparent)]
+pub struct UpdatePledgeTotalParams {
+    pub pledge_delta: TokenAmount,
+}
```

Comment:
Can we not keep using `TokenAmount`? I'm asking because the transparent thing might be even more confusing.

Reply:
We can -- this was a drive-by fix to make things conform to the pattern @anorth and i agreed on (which was to always have a type for params and returns, but serialize transparently where possible).

I can revert :woman_shrugging:, but I'd rather move towards consistency.

Reply:
Ok, fair enough.